mysql-test/r/percona_innodb_fake_changes.result

```diff
@@ -615,4 +618,60 @@ rename table t2 to t1;
 ERROR 42000: This version of MySQL doesn't yet support 'ALTER TABLE'
 set innodb_fake_changes = 0;
 drop table t2;
+create table t1 (i int) engine=innodb;
+insert into t1 values (1), (2);
+set session innodb_fake_changes = 1;
```

Comment:
We have include/start_fake_changes.inc / stop_fake_changes.inc that would add many checks that no writes happened, but they aren't usable in every case. Have you checked them?